tests/hmm/hmumu_utils.py

```diff
@@ -337,6 +338,14 @@ def default_weight(n):
     #    puid_weights = get_puid_weights(jets_passing_id, passed_puid, puidreweighting, dataset_era, parameters["jet_puid"], use_cuda)
     #    weights_individual["jet_puid"] = {"nominal": puid_weights, "up": puid_weights, "down": puid_weights}
 
+    selected_jets_pt_id = ( jets.pt > parameters["jet_pt_subleading"][dataset_era]) & (selected_jets_id)
```

Comment:
I would suggest to put the jet pt cut instead in [this line](https://github.com/jpata/hepaccelerate-cms/pull/67/files#diff-7c681c3161b87061604d79dc8695e4ffR1699) as a mask, so you keep all the cuts in one place and in a similar implementation. You can also propagate the cut values from `analysis_parameters`, so they can all be seen from one place, without going deep in the code.

Reply:
Yes, this was changed.